spec/client_spec.rb

```diff
@@ -659,6 +659,12 @@ def loop_times(times)
     expect(resp['error']).to include 'invalid SQS url'
   end
 
+  it 'check_sns with an invalid queue url should fail' do
```

Comment:
Looks like a copy-paste mistake?

```suggestion
  it 'check_sns with an invalid topic arn should fail' do
```

Reply:
thanks!